policy/pa_test.go

```diff
@@ -305,11 +295,12 @@ func TestWillingToIssue_Wildcards(t *testing.T) {
 	}
 }
 
-// TestWillingToIssueWildcards tests that more than one rejected identifier
+// TestWillingToIssue_Wildcard tests that more than one rejected identifier
```

Comment:
This test doesn't test attempting to issue for a wildcard domain. _That_ is left to `TestWillingToIssue_Wildcards` so this feels improperly named.